src/antsibull/cli/antsibull_docs.py

```diff
@@ -322,7 +324,10 @@ def parse_args(program_name: str, args: List[str]) -> argparse.Namespace:
 
     if '--ansible-base-cache' in args:
         flog.warning('The CLI parameter, `--ansible-base-cache` has been renamed to'
-                     ' `--ansible-base-source.  Please use that instead')
+                     ' `--ansible-core-source.  Please use that instead')
+    if '--ansible-base-source' in args:
+        flog.warning('The CLI parameter, `--ansible-base-source` has been renamed to'
+                     ' `--ansible-core-source.  Please use that instead')
```

Comment:
Looks like a missing ` here?
```suggestion
                     ' `--ansible-core-source`.  Please use that instead')
```

Reply:
Yes, also the same a couple of lines above. Thanks for spotting this!